plugins/com.google.cloud.tools.eclipse.appengine.ui/about.properties

```diff
@@ -0,0 +1,8 @@
+blurb=Google Cloud Tools for Eclipse\n\
+Version: {featureVersion}\n\
+\n\
+Copyright 2016-2017 Google Inc.\n\
+\n\
+Cloud Tools for Eclipse is a Google-sponsored plugin that adds \
+support for the App Engine standard environment to the Eclipse IDE.\n\
```

Comment:
We could use a more general text here instead of this narrow scope which will need to be updated whenever we add Flex, GKE, GCE. Maybe `support for the Google Cloud Platform environments`, or something along these lines?